spec/system/admin/payments_stripe_spec.rb

```diff
@@ -64,7 +64,6 @@
             fill_in_card_details_in_backoffice
             click_button "Update"
 
-            expect(page).to have_link "StripeSCA"
```

Comment:
Why removing this ?

Reply:
I thought this was redirecting to the edit page without JS. I misread this. When I changed the JS code it started rendering the new payment page on error, where before it redirected to the payment list. I'll change the controller to always render the payment list.